pages/index.js

```diff
@@ -1,63 +1,165 @@
-import { exercises as initialExercises } from "@/lib/exercises";
-import { useState } from "react";
 import {
-  StyledFlexWrapper,
-  ExerciseList,
-  StyledFilterButton,
-  StyledFilterWrapper,
-} from "/styledComponents.js";
-import Exercise from "../components/Exercise/Exercise";
-import FilterSection from "@/components/FilterSection/FilterSection";
+  motivationalQuotesBad,
+  motivationalQuotesGood,
+  motivationalQuotesOkay,
+} from "@/lib/quotes";
+import { useState } from "react";
+import { useEffect } from "react";
+import Image from "next/image";
+import styled from "styled-components";
 
 export default function HomePage() {
-  const [exercises, setExercises] = useState(initialExercises);
-  const [filters, setFilters] = useState([]);
-  const [filterMode, setFilterMode] = useState(false);
-
-  function filterExercises(exercises, filters) {
-    const filteredArray = exercises.filter((exercise) => {
-      return filters.every((filter) => exercise.muscleGroups.includes(filter));
-    });
-    return filteredArray;
-  }
+  const quotes = [
+    ...motivationalQuotesBad,
+    ...motivationalQuotesGood,
+    ...motivationalQuotesOkay,
+  ];
+
+  const [currentQuote, setCurrentQuote] = useState({
+    quote: "Loading...",
+    author: "",
+  });
+  const [moodMode, setMoodMode] = useState(null);
 
-  function onAddFilter(newFilter) {
-    setFilters([...filters, newFilter]);
+  useEffect(() => {
+    setCurrentQuote(quotes[getRandomIndex(quotes.length)]);
+  }, []);
+
+  function getRandomIndex(arrayLength) {
+    return Math.floor(Math.random() * arrayLength);
   }
 
-  function handleDisableFilter(deleteFilter) {
-    const newFiltersArray = filters.filter((filter) => filter !== deleteFilter);
-    setFilters(newFiltersArray);
+  function handleRandomQuote() {
+    setCurrentQuote(quotes[getRandomIndex(quotes.length)]);
   }
```

Comment:
You can combine these two if you want and therefore also simplify your `handleMood` function afterwards:

```js
function getRandomQuote(quotesArray) {
    return quotesArray[Math.floor(Math.random() * quotesArray.length)];
  }

  function handleMood(event, moodLevel) {
    event.preventDefault();
    if (moodLevel <= 3) {
      setCurrentQuote(getRandomQuote(motivationalQuotesBad));
      setMoodMode("Feeling off today? Don’t sweat it, you’ll bounce back!");
    } else if (moodLevel <= 7) {
    ...
    } else {
     ...
    }
  }
```